### DISCUSSION TITLE
New changes on branch are missed in PR

### DISCUSSION THREAD

**Discussion by chaoticlonghair** [community/community#65258] [Pull Requests]
### Select Topic Area

Bug

### Body

Opened a PR with a branch, then pushed another commit into that branch.

Bug:
The new commit was shown in the branch, but it wasn't shown in the PR.

![image](https://github.com/community/community/assets/30993023/6b1bc7c8-8ec5-4bf4-846d-3eed3b9312d5)

**Comment by leikang123** [comment]
You can delete the submission of this branch and resubmit the branch. When submitting, you need to check whether you are in the new branch, and then call to see

**Reply by chaoticlonghair** [reply]
The author reopen a new PR with a new branch, every thing is correct.

And, in the old PR, the "New changes since you last viewed" appeared after half hour.

But I still think it's a bug.

**Comment by leikang123** [comment]
I just tried it, and it is normal. After the PR submitted by the branch is displayed on the main branch, refresh it. Is there any other problem?

**Reply by chaoticlonghair** [reply]
Yes, it's normal now.

But it became normal after more than half hour.
In my screenshot, there is a timestamp, it was "22 minutes ago".

So, I still think it's a bug.
